BP/common/traits/BP_traits.txt

```diff
@@ -156,6 +164,9 @@ corrupt = {
 
 legendary_leader = {
 	index = 405
+	
+	# shown_in_ruler_designer = no # Right?
```

Comment:
It is only meant to be used for a limited amount of characters in history, so yes

Reply:
Done

Reply:
Should be "now not" not "now only" but eh